README.md

```diff
@@ -5,15 +5,24 @@
 grunt mocha istanbul task
 ==============
 
-[Mocha](http://visionmedia.github.com/mocha/) reporter to generate coverage report of [istanbul](http://gotwarlost.github.com/istanbul/) instrumented code, for grunt
+[Mocha](https://mochajs.org) reporter to generate coverage report of [istanbul](http://gotwarlost.github.com/istanbul/) instrumented code, for grunt
 This doesn't force you to use PhantomJS, or instrument code for server or client-side.
 
 Install
 ==============
 
-1. Install it using `npm install grunt-mocha-istanbul --save-dev`
-2. It needs `mocha`, `grunt` and `istanbul` to be installed locally on your project (aka, having them in your devDependencies)
-3. Call inside Gruntfile.js `grunt.loadNpmTasks('grunt-mocha-istanbul')`
+1. Install peer dependencies using: `npm install grunt mocha istanbul --save-dev`

```

Comment:
To avoid confusion with people who expects peer dependencies to be installed automatically and who may still be on an npm-version that does so – perhaps change the wording to "needed dependencies" rather than "peer"?